Patches/T's Samurai Faction/T_Samurai_Apparel.xml

```diff
@@ -0,0 +1,288 @@
+
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[
+                defName="TSFApparel_HaoriJacket" or
+                defName="TSFApparel_Yukata"
+                ]/costStuffCount</xpath>
+            <value>
+                <costStuffCount>60</costStuffCount>
+            </value>
+        </li>
+
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[defName="TSFApparel_Kimono"]/costStuffCount</xpath>
+            <value>
+                <costStuffCount>80</costStuffCount>
+            </value>
+        </li>
+
+<!--Armor-->
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[
+                defName="TSFApparel_AshigaruArmor" or
+                defName="TSFApparel_AshigaruHelmet" or
+                defName="TSFApparel_SamuraiArmor" or
+                defName="TSFApparel_SamuraiHelmet" or
+                defName="TSFApparel_DaimyoHelmet"]/stuffCategories</xpath>
+            <value>
+                <stuffCategories>
+                    <li>Steeled</li>
+                </stuffCategories>      
+            </value>
+        </li>
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[
+                defName="TSFApparel_AshigaruArmor" or 
+                defName="TSFApparel_SamuraiArmor"]/apparel/layers</xpath>
+            <value>
+                <layers>
+                    <li>Shell</li>             
+                </layers>
+            </value>
+        </li>
+
+<!--Ashigaru-->
+    <!--Armor-->      
+        <li Class="PatchOperationAdd">
+            <xpath>Defs/ThingDef[defName="TSFApparel_AshigaruArmor"]/statBases</xpath>
+            <value>
+                <Bulk>40</Bulk>
+                <WornBulk>0</WornBulk>
+            </value>
+        </li>
+
+        <li Class="PatchOperationReplace">
+			<xpath>Defs/ThingDef[defName = "TSFApparel_AshigaruArmor"]/statBases/StuffEffectMultiplierArmor</xpath>
+			<value>
+				<StuffEffectMultiplierArmor>1.6</StuffEffectMultiplierArmor>
+			</value>
+		</li>
+        
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[defName="TSFApparel_AshigaruArmor"]/costStuffCount</xpath>
+            <value>
+                <costStuffCount>110</costStuffCount>
+            </value>
+        </li>
+
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[defName="TSFApparel_AshigaruArmor"]/statBases/Mass</xpath>
+            <value>
+                <Mass>9</Mass>
+            </value>
+        </li>
+
+        <li Class="PatchOperationRemove">
+            <xpath>Defs/ThingDef[defName = "TSFApparel_AshigaruArmor"]/equippedStatOffsets</xpath>
+        </li>
+
+    <!--Helmet-->
+        <li Class="PatchOperationReplace">
+            <xpath>Defs/ThingDef[defName = "TSFApparel_AshigaruHelmet"]/statBases/StuffEffectMultiplierArmor</xpath>
+            <value>
+                <StuffEffectMultiplierArmor>2</StuffEffectMultiplierArmor>
```

Comment:
Reduce to 1.8, 2 is generally for higher quality helmets of this era.

Reply:
Ye, I made that because I was a bit unsure about when looking at the CE bascinet (which i'm not sure what it's real purpose is)